action.yml

```diff
@@ -1,20 +1,23 @@
-name: Validate Tables
-description: Validates the presence of tables in the Deephaven server
+name: Assert Tables
+description: Asserts the presence of tables in a Deephaven server
 inputs:
   table-names:
     description: A string containing a comma separated list of table names to check
     required: true
   host:
-    description: The host name of the Deephaven instance
-    required: false
-    default: envoy
-  project-name:
-    description: The docker-compose project name to bundle separate docker-compose files
+    description: The host name or IP address of the Deephaven instance
+    required: true
+  docker-compose-project-name:
```

Comment:
I still don't think we need this in the final design, but we can talk about it.

Reply:
Based on our conversation let's leave this open for now. If the composite action works we could remove this, or if we add a separate test-build action too

Reply:
Removed! No need for the parameter anymore with the new `composite` action